tenable/io/v3/vm/schema.py

```diff
@@ -0,0 +1,33 @@
+import copy
+
+from marshmallow import Schema, fields, post_dump
+from marshmallow import validate as v
+
+
+class ScannerEditSchema(Schema):
+    """
+    Schema for edit functions in scanners.py
+
+    Args:
+
+    """
+
+    id = fields.Int(required=True)
+    force_plugin_update = fields.Bool()
+    force_ui_update = fields.Bool()
+    finish_update = fields.Bool()
+    registration_code = fields.Str()
+    aws_update_interval = fields.Int()
+
+    @post_dump
+    def post_serialization(self, data, **kwargs):  # noqa PLR0201 PLW0613
+        data_dict = copy.deepcopy(data)
+
+        for key, value in data.items():
```

Comment:
Why not use `map` or another function to iterate? I think that might be cleaner here than a `for` loop with nested logic, and an extra dict to store.

Reply:
- Updated post_dump to use map function there.
- post_dump is used as per previous code behaviour the payload was transformation was following
```
Input: 
 {
        "force_plugin_update": True,
        "force_ui_update": False,
        "finish_update": True,
        "registration_code": "random_code",
        "aws_update_interval": 60,
 }

Payload:

   {
        "force_plugin_update": 1,
        "finish_update": 1,
        "registration_code": "random_code",
        "aws_update_interval": 60,
    }
```
- False params were not sent as part of the payload and using BoolInt class will not provide us the same behaviour hence updated post_dump method accordingly